contracts/Outcome.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity ^0.5.11;
+pragma experimental ABIEncoderV2;
+
+library Outcome {
+    //An outcome is a set of asset outcomes.
+    // Outcome = AssetOutcome[]
```

Comment:
This comment implies that an outcome can be a mix of allocations and guarantees. Is that the case? Or should an outcome only contain either allocations or guarantees?